libs/dh/message-archive/feature-log-search/src/lib/dh-message-archive-log-search.component.html

```diff
@@ -55,86 +46,62 @@ <h1>{{ transloco('title') }}</h1>
         />
       </watt-form-field>
 
-      <watt-form-field size="normal">
-        <watt-label class="watt-text-m">{{ transloco('role') }}</watt-label>
-
-        <select
-          #searchInputRole
-          wattInput
-          [(ngModel)]="this.searchCriteria.senderRoleType"
-          name="role"
-        >
-          <option selected [ngValue]="null">
-            {{ transloco('ChooseRole') }}
-          </option>
-          <option
-            *ngFor="let roleType of this.roleTypes | keyvalue"
-            [ngValue]="roleType['key']"
-            value="{{ roleType['value'] }}"
-          >
-            {{ roleType['key'] + ' - ' + roleType['value'] }}
-          </option>
-        </select>
-      </watt-form-field>
-
       <watt-form-field size="normal">
         <watt-label class="watt-text-m">{{
-          transloco('messageId')
+          transloco("receiverId")
         }}</watt-label>
 
         <input
-          #searchInputMessageId
-          [(ngModel)]="this.searchCriteria.messageId"
+          #searchInputReceiverId
+          [(ngModel)]="this.searchCriteria.receiverId"
           wattInput
           type="text"
-          name="messageId"
+          name="receiverId"
         />
       </watt-form-field>
 
       <watt-form-field size="normal">
         <watt-label class="watt-text-m">{{
-          transloco('receiverId')
+          transloco("messageId")
         }}</watt-label>
 
         <input
-          #searchInputReceiverId
-          [(ngModel)]="this.searchCriteria.receiverId"
+          #searchInputMessageId
+          [(ngModel)]="this.searchCriteria.messageId"
           wattInput
           type="text"
-          name="receiverId"
+          name="messageId"
         />
       </watt-form-field>
 
       <watt-form-field size="normal">
         <watt-label class="watt-text-m">{{
-          transloco('businessReason')
+          transloco("businessReason")
         }}</watt-label>
+        <watt-dropdown
+          [(ngModel)]="this.searchCriteria.processType"
+          [formControl]="processTypeFormControl"
+          [options]="processTypeFormFieldOptions"
+          [placeholder]="transloco('ChooseBusinessReason')"
+          [noOptionsFoundLabel]="'-'"
+        >
+        </watt-dropdown>
+      </watt-form-field>
 
-        <select
-          #searchInputBusinessReasonCode
+      <!-- Reference id -->
+      <watt-form-field size="normal">
```

Comment:
suggestion: the `watt-checkbox` component is not designed to be used inside `watt-form-field`. You can remove the `watt-form-field` wrapper.